react-web/src/pages/auditing/reportUpload/ReportUpload.tsx

```diff
@@ -0,0 +1,317 @@
+            id="logo"
+            {...form.register("logo")}
+          />
+
+          <Input
+            label="Discord"
+            type="text"
+            id="discord"
+            {...form.register("discord")}
+          />
+
+          <Input
+            label="Twitter"
+            type="text"
+            id="twitter"
+            {...form.register("twitter")}
+          />
+
+          <div className="separator-label">Audit Report</div>
+          {/* <Upload
+            isMultiple={false}
+            highlightText="Upload a PDF"
+            uploadFiles={(file) => {
+              console.log("file", file);
+              setFiles(file);
+            }}
+            showPreview
+            tooltipText={`Please upload a ${SUPPORTED_FORMATS.join(",")} file within 5MB size`}
+            maxFileSize={FILE_SIZE}
+            acceptedTypes={SUPPORTED_FORMATS.join(",")}
+            uploadedFiles={files}
+            required={true}
+            className="bordered"
+            onClick={(files) => console.log(files)}
+            name="auditReport"
+            showDefaultError
+          /> */}
+          <TextArea
+            placeholder="Report URLs"
+            required={true}
+            maxRows={2}
+            tooltipText="Enter comma(,) separatated JSON/PDF URLs or IPFS link corresponding to the report"
+            {...form.register("reportURL")}
+          />
+
+          <div className="separator-label">DAPP Script</div>
+          <div className="relative">
+            <div className="absolute action-button addScript-btn">
+              <Button
+                displayStyle="primary-outline"
+                size="small"
+                buttonLabel="+ Add Script"
+                type="button"
+                disabled={
+                  !!form?.formState.errors?.[fieldArrayName] || // disable button if errors associated with dynamic form exists
+                  form
+                    .getValues(fieldArrayName)
+                    ?.some(
+                      (field: { scriptHash: any; contactAddress: any }) =>
+                        !field?.scriptHash || !field?.contactAddress
+                    ) // prevent addition of new script boxes if the required field is empty
+                }
+                onClick={() => { addNewDappScript() }}
+              />
+            </div>
+
+            {fields.map((field, index) => (
+              <DAPPScript
+                key={field.id}
+                remove={remove}
+                value={field}
+                index={index}
+              />
+            ))}
+          </div>
+
+          <div className="button-wrapper">
+            <Button
+              type="button"
+              displayStyle="secondary"
+              buttonLabel={"Cancel"}
+              onClick={() => {
+                form.reset();
+                navigate(-1);
+              }}
+            />
+
+            <Button
+              disabled={!form.formState.isValid}
```

Comment:
I can't make it valid, not sure what I'm doing wrong. I've filled all fields, even the non mandatory ones, and I have no error displayed in the form.

Reply:
@RSoulatIOHK Please recheck the branch for these issues and workflow in general